plugins/modules/meraki_ms_switchport.py

```diff
@@ -471,6 +471,12 @@ def clear_vlan(params, payload):
     return payload
 
 
+def clear_polices(params, payload):
+    if params["access_policy_type"] is None:
+        payload["accessPolicyType"] = "Open"
```

Comment:
I think we should be careful about using `None` here. It's a bad sentinel value because there is no default and if someone doesn't include it, it will change their environment. The modules are developed under the assumption they aren't the single source of truth.

Reply:
All right! I will think about that. BTW it is good case.

Reply:
Thinking this through. If you set it to open does it automatically reset the policy?

Reply:
It is not reset a policy. It enable to change the type from access to trunk. Because the trunk does not allow to set the custom policy.
I would like to show it but I have problems with Sandbox :(

Reply:
I'm concerned about leaving this as `is None`. The modules are written so that if a playbook author doesn't include a parameter in a task, it doesn't overwrite it by default. If I understand how this works correctly, it's going to set the access policy type to open if they don't include that parameter, which to me is destructive in an implicit sense. I haven't experimented with this situation very much, but is there a reason `Open` doesn't work for this purpose?

Reply:
Without that code the example test for changing the port from `access with access_policy` to `trunk`. That function firstly change the `access_policy_type` to `Open`. After that change it is enable to change port from `access` to `trunk`. I will more describe it above.